Team_Roster/app/__init__.py

```diff
@@ -9,4 +9,4 @@
 migrate = Migrate(app, db)
 
 
-from app import routes
\ No newline at end of file
+from Team_Roster.app import routes
```

Comment:
Just curious, what is your reason for making this change?

Reply:
My code was having an issue finding the path to app, but specifying the directory helped